packages/flutter_tools/templates/app/android-kotlin.tmpl/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.4.1'
```

Comment:
Same here - if we keep this in sync we should hopefully catch failures more easily and test more things (like consuming plugins)